server/services/user.js

```diff
@@ -10,10 +10,15 @@ const DEVICE_SCHEMA = Joi.object().keys({
   contactInformation: Joi.string()
 });
 
+function exists(userId) {
+  return User.exists(userId);
+}
+
 function createUser(userObject) {
   const userSchema = Joi.object().keys({
     name: Joi.string().required(),
-    email: Joi.string().email().required()
+    email: Joi.string().email().required(),
+    isSysAdmin: Joi.boolean()
```

Comment:
Nope, it just means that the updateUser method can be used to make that update. There is middleware that stops non admins from creating admins